docker/docker-dev-custom-cfg-kafka/metrictank.ini

```diff
@@ -385,7 +385,7 @@ disable-initial-host-lookup = false
 [memory-idx]
 enabled = false
 # enables/disables querying based on tags
-tag-support = false
+tag-support = true
```

Comment:
was that committed intentionally? or did you only change that for testing?

Reply:
That was for testing, forgot to revert it before commit